roboteam_ai/src/interface/widget.cpp

```diff
@@ -289,6 +291,24 @@ bool Visualizer::robotIsSelected(roboteam_msgs::WorldRobot robotToCheck) {
     return false;
 }
 
+void Visualizer::drawIntercept(QPainter &painter, std::vector<std::pair<rtt::Vector2, QColor>> points) {
+    for (int j = 0; j <5; ++ j) {
```

Comment:
magic number 5

Reply:
Again, length of vector of visualization points, so not a magic number.